src/data/courseDependenciesData.js

```diff
@@ -959,61 +959,34 @@ const courseDependencyData = (courseSlug) =>
             slug: 'the-beginner-s-guide-to-react',
           },
         ],
-        courseProject: {
```

Comment:
Adding a portfolio project block that points to the custom landing page is confusing. I think both now both course pages can exist without linking to each other.